src/models/FieldLayoutForm.php

```diff
@@ -69,6 +69,11 @@ public function render(bool $showFirst = true): string
                     'id' => $id,
                     'layout-tab' => $tab->getUid(),
                 ],
+                'role' => 'tabpanel',
+                'tabindex' => '0',
+                'aria' => [
+                    'labelledBy' => 'tab-' . $tab->getId(),
```

Comment:
This seems hacky - better way to specify the ID of the tab button that activates the tabpanel?
<img width="360" alt="Screenshot showing accessibility information of tab button" src="https://user-images.githubusercontent.com/17525979/154569236-f1a8be6f-2b0b-4e7e-8406-96c8d71f6516.png">

Reply:
This is maybe a little bit cleaner: d14fa747d38fb0613545dab5cd2d0ab21863014f